tests/test_frontend/test_feedback/test_collect_feedback.py

```diff
@@ -14,11 +14,9 @@ def test_collect_full_feedback(self):
 
             new_page = new_page_info.value
             new_page.get_by_label("Very dissatisfied").click()
```

Comment:
I changed this line to:
```suggestion
            new_page.get_by_label("Very dissatisfied").check()
```
which seems to have worked. Otherwise we'd be removing the check on the full feedback item.